src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -18,19 +18,19 @@ public class HelpCommand extends Command {
 
     // All commands summary
     public static final String ALL_COMMANDS_MESSAGE = "Here are all the valid commands:\n"
-            + "add n/CLIENT a/ADDRESS p/PHONE e/EMAIL [t/TAG]\n"
-            + "create INDEX n/NAME p/NUMBER e/EMAIL [t/TAG]\n"
-            + "buy INDEX q/QUANTITY g/GOODS price/PRICE\n"
-            + "sell INDEX q/QUANTITY g/GOODS price/PRICE\n"
-            + "edit INDEX [n/NAME] [a/ADDRESS] [p/PHONE] [e/EMAIL] [t/TAG]\n"
-            + "view INDEX\n"
-            + "delete INDEX\n"
-            + "find KEYWORD\n"
-            + "exit\n"
-            + "help\n"
-            + "list\n"
-            + "user_guide\n"
-            + "clear";
+            + "1. add n/CLIENT a/ADDRESS p/PHONE e/EMAIL [t/TAG]\n"
+            + "2. buy INDEX q/QUANTITY g/GOODS price/PRICE\n"
+            + "3. sell INDEX q/QUANTITY g/GOODS price/PRICE\n"
+            + "4. edit INDEX [n/NAME] [a/ADDRESS] [p/PHONE] [e/EMAIL] [t/TAG]\n"
```

Comment:
This edit command now includes another tag after INDEX which is either m/client m/transaction, or m/remark. I can edit this as well

Reply:
no worries, will push another commit to update!